test/functional/rpc_net.py

```diff
@@ -167,6 +167,8 @@ def test_getnettotals(self):
         self.nodes[0].ping()
         self.wait_until(lambda: (self.nodes[0].getnettotals()['totalbytessent'] >= net_totals_before['totalbytessent'] + 32 * 2), timeout=1)
         self.wait_until(lambda: (self.nodes[0].getnettotals()['totalbytesrecv'] >= net_totals_before['totalbytesrecv'] + 32 * 2), timeout=1)
+        self.wait_until(lambda: (self.nodes[0].getnettotals()['bytesrecv_per_msg'].get('pong', 0) >= net_totals_before['bytesrecv_per_msg'].get('pong', 0) + 32), timeout=1)
+        self.wait_until(lambda: (self.nodes[0].getnettotals()['bytessent_per_msg'].get('ping', 0) >= net_totals_before['bytessent_per_msg'].get('ping', 0) + 32), timeout=1)
```

Comment:
add functional test coverage to test the lifetime of a connection

Reply:
the baseline for bytes per message bytes should be deterministic